### ISSUE TITLE
If `pkonly` is disabled, then players without `canbehead` could still behead other players.  (original 3.x behavior)

### ISSUE THREAD

**Issue by crashdemons** [crashdemons/PlayerHeads#6]
From the original code:

```
        if (plugin.configFile.getBoolean("pkonly") && ((killer == null) || (killer == player) || !killer.hasPermission("playerheads.canbehead"))) {
            return;
        }
```
So if `pkonly` is disabled, then a player can behead regardless of their permission - is this supposed to be intended behavior?

**Comment by crashdemons** [comment]
On 4.x, this is changed by e1f3fd58ae91c556aa4fce3c9598884f7cc15fe9 and 50f613b096aa2a86402b3fea634960bc1c7b03a0 but I need to test it.

On 3.x this is unchanged.

**Comment by crashdemons** [comment]
Fixed in 4.2.2-alpha release.  Orginal behavior is kept in 3.12-shimmed1.13
